tests/page-objects/search/search.wdio.page.js

```diff
@@ -0,0 +1,25 @@
+const searchBox = () => $('input#freetext');
```

Comment:
I like the addition of the search page. There's some duplication in the `contacts.wdio.page.js` now because of this though. It would be better if we could get rid of the duplicates.

Reply:
```
const searchBox = () => $('#freetext');
const searchButton = () => $('#search');
```
```
const search = async (query) => {
  await (await searchBox()).setValue(query);
  await (await searchButton()).click();
};
```

Reply:
No longer part of this PR